index.js

```diff
@@ -30,20 +30,14 @@ module.exports = {
 
     if (options.includeTimezone) {
       this.import(
-        {
-          development: 'vendor/moment-timezone/tz.js',
-          production: 'vendor/moment-timezone/tz.min.js'
-        },
+        'vendor/moment-timezone/tz.js',
         { prepend: true }
       );
     }
 
     if (typeof options.includeLocales === 'boolean' && options.includeLocales) {
       this.import(
-        {
-          development: 'vendor/moment/min/moment-with-locales.js',
-          production: 'vendor/moment/min/moment-with-locales.min.js'
-        },
+        'vendor/moment/moment-with-locales.js',
```

Comment:
This should be `'vendor/moment/min/moment-with-locales.js'` I think

Reply:
That's it ! 🥇

Reply:
It's been a while, but IIRC including those minified was exactly what was breaking before. Maybe it's fixed upstream now, I have no idea. Happy to take a PR as long as it's tested with the latest versions of ember-cli and ember-cli-terser.

Reply:
not sure if it's minified or not, but the path is wrong, there's nothing at the path current specified in 3.8!

Reply:
These paths have changed within minor versions of moment in the past.  Figured I'd call that out for anyone looking at a solution.

Reply:
it's not a problem that the path changed, it's a problem that the build process is broken because it references the wrong path

Reply:
I see - @mehulkar feel free to open a PR once you verify.

Reply:
@jasonmit I have created a PR to fix this issue.
https://github.com/jasonmit/ember-cli-moment-shim/pull/196